addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -68,6 +68,7 @@
  * sent to one of the channels.
  *
  * @author Patrick Hecker - Initial contribution
+ * @author Johannes Einig - handleUpdate deactivated (deprecated); handleCommand debug info added
```

Comment:
I would not say that the following description explains the core of your changes:

> handleUpdate deactivated (deprecated); handleCommand debug info added

Maybe mention the caching